pkgs/top-level/python-packages.nix

```diff
@@ -13057,6 +13056,7 @@ let
     };
 
     propagatedBuildInputs = with self; [
+      requests
```

Comment:
It's a bit confusing, but `requests` means 1.x branch and `requests2` means 2.x branch